testsuite/MDAnalysisTests/transformations/test_rotate.py

```diff
@@ -102,8 +106,10 @@ def test_rotateby_atomgroup_com_nopbc(rotate_universes):
     angle = 90
     selection = trans_u.residues[0].atoms
     center_pos = selection.center_of_mass()
-    matrix = rotation_matrix(np.deg2rad(angle), vector, center_pos)[:3, :3]
-    ref.positions = np.dot(ref.positions, matrix)
+    matrix = rotation_matrix(np.deg2rad(angle), vector, center_pos)
+    rotation = matrix[:3, :3]
+    translation = matrix[:3, 3]
+    ref.positions = np.dot(ref.positions, rotation) + translation
```

Comment:
Can you use red.transform for the test. It’s an atom group method that accepts the 4x4 matrix for translation and rotation.

Reply:
Really it does that? I never use the combined method much myself. Did you check visually if the alignment is correct like this now or before?